include/deal.II/distributed/tria_base.h

```diff
@@ -160,6 +167,7 @@ namespace parallel
       std::vector<unsigned int> n_locally_owned_active_cells;
       types::global_dof_index   n_global_active_cells;
       unsigned int              n_global_levels;
+      std::vector<unsigned int> ghost_neighbors;
```

Comment:
I know the other members aren't documented either, but could you add something for the new variable at least?